django/thunderstore/core/settings.py

```diff
@@ -258,6 +258,11 @@ def load_db_certs():
 
 USE_TZ = True
 
+# Sessions
+
+# Session cookie used by React components during Django React transition
+SESSION_COOKIE_HTTPONLY = False
```

Comment:
The default should be False, was this not the case?

Reply:
The default is `True` because the session cookie isn't accessed by JavaScript in most sites and helps reduce the effect of an XSS

Reply:
Ah, looks like I got this confused with `CSRF_TOKEN_HTTPONLY`.